### PR TITLE
Ossfuzz corp/v4

### PR DESCRIPTION
**Pull request by victorjulien**

### CODE REVIEW

File: qa/run-ossfuzz-corpus.sh [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+ls src/tests/fuzz/fuzz_*.c | sed 's/\.c//' | while read ftarget
+do
+    target=$(basename $ftarget)
+    echo "target $target"
+    #download public corpus
+    rm -f public.zip
+    wget --quiet "https://storage.googleapis.com/suricata-backup.clusterfuzz-external.appspot.com/corpus/libFuzzer/suricata_$target/public.zip"
+    rm -rf corpus_$target
+    unzip -q public.zip -d corpus_$target
+    #run target on corpus. Don't fail CI if the target fails.
```
Comment (catenacyber) [inline qa/run-ossfuzz-corpus.sh:11]: Why not fail the CI if the target fails ?
(because we need to fix some bugs first ?)

**Comment by catenacyber** [comment]
Will you want to cherry-pick https://github.com/OISF/suricata/pull/5613/commits/f520bc88eff8f9100a49089143a39d716522274e ?

File: qa/run-ossfuzz-corpus.sh (hunk 1 above)
Comment (victorjulien) [reply qa/run-ossfuzz-corpus.sh:11]: Right now I want to understand how we can max out the coverage, and it seems its not yet working as expected. With fuzz enabled we have only a minimal amount of extra code, yet coverage drops? I think when I'm confident the cov is handled correctly I want to either update the test to fail or add a second test that can fail

Comment (victorjulien) [reply qa/run-ossfuzz-corpus.sh:11]: The math doesn't add up for me, maybe I'm overlooking something. I seems like cov for the unittests and verify flags didn't change, and we added a new flag with some cov. So how did things go down?

Comment (catenacyber) [reply qa/run-ossfuzz-corpus.sh:11]: Can you visualize the diff in coverage ?

Comment (victorjulien) [reply qa/run-ossfuzz-corpus.sh:11]: Yeah you can look here https://app.codecov.io/gh/OISF/suricata/compare/5766

Comment (catenacyber) [reply qa/run-ossfuzz-corpus.sh:11]: Indeed, it does not seem consistent and now buggy with error "Missing base report." (was ok two minutes ago)

**Comment by victorjulien** [comment]
> Will you want to cherry-pick [f520bc8](https://github.com/OISF/suricata/commit/f520bc88eff8f9100a49089143a39d716522274e) ?

I tried in #5768, but it leads to many build failures.

**Comment by catenacyber** [comment]
This one seems to work 
https://github.com/catenacyber/suricata/pull/2/commits/5cf523ed14c7fc47a22428d54b666c10a51e294f

**Comment by catenacyber** [comment]
Replaced by https://github.com/OISF/suricata/pull/5768 ?

**Pull request closed without merging**